Makefile

```diff
@@ -17,7 +17,7 @@ SRC ?= github.com/kubernetes-sigs/cri-tools
 TAG ?= ${GITHUB_ACTION_TAG}
 
 ifeq ($(TAG),)
-TAG := v1.26.1$(BUILD_META)
+TAG := v1.31.1$(BUILD_META)
```

Comment:
Only change this to the lastest 1.28 release, otherwise we lock ourselves out of releasing for older stuff.
```suggestion
TAG := v1.28.14$(BUILD_META)
```

Reply:
GH will still build newer tags, this can be considered a "floor" for the tags

Reply:
When I didn't change that though, the output of the GHA showed 1.26.1 even though I set TAG in the Dockerfile